app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,9 @@
   </head>
 
   <body>
+    <% if current_user %>
```

Comment:
On every view you want this title :thinking: 
Better if you set up a flash message after login only, no ?